tests/linearizability/model/history.go

```diff
@@ -382,19 +382,12 @@ type History struct {
 	failed []porcupine.Operation
 }
 
-func (h History) Merge(h2 History) History {
-	result := History{
-		successful: make([]porcupine.Operation, 0, len(h.successful)+len(h2.successful)),
-		failed:     make([]porcupine.Operation, 0, len(h.failed)+len(h2.failed)),
-	}
-	result.successful = append(result.successful, h.successful...)
-	result.successful = append(result.successful, h2.successful...)
-	result.failed = append(result.failed, h.failed...)
-	result.failed = append(result.failed, h2.failed...)
-	return result
+func (h *History) Merge(h2 History) {
```

Comment:
nit: LGTM, but I wonder what was the reason for original [implementation](https://github.com/etcd-io/etcd/commit/d8da98c49fbb6b61faf805419c5356a05f43793f#diff-6c2dd0f0e7caef34835e61886c9988d902e3184c2433b30dc51686d30e7538c1) using value receiver? cc @serathius

Reply:
Value receiver implies that value is not mutated.